src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -41,6 +43,7 @@ function ProviderSelectionField({
     0,
     providersListLength,
   );
+  const providersListEmpty = currentlyShownProvidersList.length === 0;
```

Comment:
Probably worth doing `currentlyShownProvidersList?.length`, it's possible that variable is undefined before the list is done being fetched.